packages/toolpad-app/src/toolpad/ToolpadHomeShell.tsx

```diff
@@ -0,0 +1,66 @@
+import {
+  Box,
+  Divider,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Stack,
+  styled,
+} from '@mui/material';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import FlexFill from '../components/FlexFill';
+import { DOCUMENTATION_URL, REPOSITORY_URL } from '../constants';
+import ToolpadShell, { ToolpadShellProps } from './ToolpadShell';
+
+const NavigationListItemButton = styled(ListItemButton)({ height: 56 });
+
+export interface HomeShellProps extends ToolpadShellProps {}
+
+export default function HomeShell({ children, ...props }: HomeShellProps) {
+  return (
+    <ToolpadShell {...props}>
+      <Stack direction="row" sx={{ height: '100%' }}>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            width: 250,
+            mt: 3,
+            ml: 2,
+            overflow: 'auto',
+          }}
+        >
+          <List>
+            <ListItem disablePadding>
+              {/* @ts-expect-error https://github.com/mui/material-ui/issues/29875 */}
+              <NavigationListItemButton component={Link} to={'/apps'}>
+                <ListItemText primary="Apps" />
```

Comment:
Should we have icons for the list items too? But probably not too important for now.

Reply:
Let's wait for the design to be finalized